kubernetes/cache/kube_cache.go

```diff
@@ -409,6 +421,18 @@ func (c *kubeCache) createGatewayInformers(namespace string) gateway.SharedInfor
 		lister.k8sreferencegrantLister = sharedInformers.Gateway().V1beta1().ReferenceGrants().Lister()
 		lister.cachesSynced = append(lister.cachesSynced, sharedInformers.Gateway().V1beta1().ReferenceGrants().Informer().HasSynced)
 		c.hasGatewayAPIStarted = true
+
+		if c.client.IsExpGatewayAPI() {
```

Comment:
Probably it would be useful to have any debug message here?

Reply:
I avoided any message there because the feature is experimental, but the debug level can be helpful.